src/env.h

```diff
@@ -1611,6 +1625,9 @@ class Environment : public MemoryRetainer {
   // a given pointer.
   std::unordered_map<char*, std::unique_ptr<v8::BackingStore>>
       released_allocated_buffers_;
+
+  // Maintains stored Blobs used with the URL.createObjectURL() API
+  std::unordered_map<std::string, StoredDataObject> data_objects;
```

Comment:
Let’s write clean code and a) split the blob stuff from the `buffer` binding into its own binding and b) add this to a `BindingData` object for the blob binding, not tack it onto the environment.